src/main/java/org/folio/service/PrefixService.java

```diff
@@ -75,4 +75,19 @@ private CompletableFuture<Void> checkPrefixNotUsed(Prefix prefix, RequestContext
         }
       });
   }
+
+  public CompletableFuture<Void> isPrefixAvailable(String prefixName, RequestContext requestContext) {
```

Comment:
a

Reply:
yeah, i agree too, but not sure if we can change existing className.
Cc:- @SerhiiNosko

Reply:
Basically we have separate tables for storing prefixes and suffixes, and we need some service to work with them using RestClient that calls storages services. After this we inject it into main PurchaseOrderHelper that accumalates all services needed to work with orders. If we avoid separate services for prefixes and suffixes - we directly need to inject RestClient into PurchaseOrderHelper that brokes abstraction levels (we don't have separate OrderService, we have only this hepler that injects all together).

Reply:
I suggest to rename method name like this : "valedatePrefixAvailable"

Reply:
@SerhiiNosko the prefix and suffix services existed before this PR, so I am open to leaving it as is. I would like TODO to be added to both classes to refactor it so that its explicit relationship to only purchase orders is considered.